packages/office-addin-debugging/src/usagedata-helper.ts

```diff
@@ -0,0 +1,21 @@
+import * as defaults from './defaults';
+import * as usageData from "office-addin-usage-data";
+const usageDataObject: usageData.OfficeAddinUsageData = new usageData.OfficeAddinUsageData(defaults.usageDataOptions);
+
+export function sendUsageDataSuccessEvent(method: string) {
+    const usageDataInfo = {
+        Method: [method],
+        Platform: [process.platform],
+        NodeVersion: [process.version],
+        Succeeded: [true]
+    }
+    usageDataObject.reportEvent(defaults.usageDataProjectName, usageDataInfo);
+}
+
+export function sendUsageDataCustomEvent(usageDataInfo: Object) {
+    usageDataObject.reportEvent(defaults.usageDataProjectName, usageDataInfo);
+}
+
+export function sendUsageDataException(method: string, error: string) {
```

Comment:
This function should take `error: string | Error` and have logic in its implementation to convert the type to a string. 

if error is an Error object, use error.message.
if error is a string, use error.
if another type, could happen anyway, I would turn this into a string like `Unexpected type: ${JSON.stringify(error)}`